src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { useState } from "react";
+
+function ArticleCard({article}){
+    return (
+        <div className="HomeArticleCard">
+            <h3>Title: {article.title}</h3>
+            <h4>Author: {article.author} </h4>
+            <h4>Topic: {article.topic}</h4>
+        </div>
```

Comment:
I'd consider displaying the image associated with the article to make your app more engaging and user-friendly 😄